taskcluster/xpi_taskgraph/parameters.py

```diff
@@ -0,0 +1,29 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from __future__ import absolute_import, print_function, unicode_literals
+
+import os
+
+import six
+from six import text_type
+from taskgraph.parameters import extend_parameters_schema
+from voluptuous import Required
+
+
+extend_parameters_schema({
+    Required("template_base_repository"): text_type,
+    Required("template_head_repository"): text_type,
+    Required("template_head_ref"): text_type,
+    Required("template_head_rev"): text_type,
+    Required("template_repository_type"): text_type,
+})
+
+
+def get_decision_parameters(graph_config, parameters):
+    parameters["template_base_repository"] = text_type(os.environ.get("TEMPLATE_BASE_REPOSITORY", ""))
```

Comment:
I don't see these being used anywhere.

But, if it is needed, I think it should be in taskgraph, not here.

Reply:
(I had actually played around a bit with loading all these parameters from the environment, with support for multiple repositories when I first extracted taskgraph, but abandoned it for compatibility/expediency at the time. It might be worthwhile to revisit doing that).

Reply:
I just removed them, and lost the info in params. These are just here for debugging purposes later, so we can determine which revision of xpi-template was used to build the graph.

I'm leaning towards putting them back in. What do you mean by putting it in taskgraph?

Reply:
Filed https://github.com/mozilla-extensions/xpi-template/issues/6 .

Reply:
Looks like we get a TinderboxPrint with the revision in the decision log. That's probably sufficient for debugging purposes.